frontend/src/shared/book-display/ShelfCarousel.css

```diff
@@ -9,20 +9,16 @@
 
 /* Styles for carousel */
 
-.container {
-    margin-top: 40px;
-    margin-left: 180px;
-}
-
 .shelf-container {
-    padding: 0px 40px;
+    margin-bottom: 40px;
 }
 
 .shelf-title {
     display: inline-block;
     font-size: 20px;
```

Comment:
Looks like we can bump the font-size up a tad, to 24px based of specs in Figma :)